### PR TITLE
Multi request client.

### PR DESCRIPTION
**Pull request by boaks**
Extend example client to start multiple connections and exchange
multiple messages.

Signed-off-by: Achim Kraus <achim.kraus@bosch-si.com>

### CODE REVIEW

File: demo-apps/sc-dtls-example-client/src/main/java/org/eclipse/californium/scandium/examples/ExampleDTLSClient.java [hunk 1]
```diff
@@ -83,7 +85,9 @@ public ExampleDTLSClient() {
             builder.setIdentity((PrivateKey) keyStore.getKey("client", KEY_STORE_PASSWORD.toCharArray()),
                     keyStore.getCertificateChain("client"), true);
             builder.setTrustStore(trustedCertificates);
+            builder.setEnableAddressReuse(false);
             dtlsConnector = new DTLSConnector(builder.build());
+            dtlsConnector.setExecutor(new StripedExecutorService(2));
```
Comment (sbernard31) [inline demo-apps/sc-dtls-example-client/src/main/java/org/eclipse/californium/scandium/examples/ExampleDTLSClient.java]: You may use builder.setConnectionThreadCount(2);

Comment (boaks) [inline demo-apps/sc-dtls-example-client/src/main/java/org/eclipse/californium/scandium/examples/ExampleDTLSClient.java]: In the meantime, yes.

File: demo-apps/sc-dtls-example-client/src/main/java/org/eclipse/californium/scandium/examples/ExampleDTLSClient.java [hunk 2]
```diff
@@ -111,15 +127,46 @@ private void test(InetSocketAddress peer) {
...
-		client.test(peer);
-		latch.await(5, TimeUnit.SECONDS);
+		client.start(peer);
+		return client;
+	}
+
+	public static void main(String[] args) throws InterruptedException {
+		long count = 0;
+		long nanos = System.nanoTime();
+		int max = 10;
```
Comment (sbernard31) [inline demo-apps/sc-dtls-example-client/src/main/java/org/eclipse/californium/scandium/examples/ExampleDTLSClient.java]: I will go for only 1 client by default.

**Comment by sbernard31** [comment]
LGTM

**Pull request merged**